pkg/sub_controller/be/be_pod.go

```diff
@@ -190,6 +190,7 @@ func (be *BeController) buildPodTemplate(src *srapi.StarRocksCluster, beconfig m
 		ImagePullSecrets:              beSpec.ImagePullSecrets,
 		NodeSelector:                  beSpec.NodeSelector,
 		HostAliases:                   beSpec.HostAliases,
+		SchedulerName:                 beSpec.SchedulerName,
```

Comment:
If `beSpec.SchedulerName` is not assigned, then `beSpec.SchedulerName` is a zero value, which is an empty string for string type. `SchedulerName `is a string type, as a field in the `PodSpec` struct, and is also an empty string by default.